pkg/controller/admissionchecks/provisioning/constants.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provisioning
+
+const (
+	ConfigKind            = "ProvisioningRequestConfig"
+	ControllerName        = "ProvisioningRequestController"
+	ConsumesAnnotationKey = "cluster-autoscaler.kubernetes.io/consume-provisioning-request"
```

Comment:
@alculquicondor @trasc IIUC, k-sigs project needs to use `x-k8s.io` domain instead of `kubernetes.io`, right?

Reply:
Not necessarily. Officially reviewed APIs, even if not part of core k8s, can use `kubernetes.io`. The GatewayAPI is one of them.

But I guess this is the current value.

@kisieland can you confirm if there any plans to change this?

Reply:
`ProvisioningRequest` API seems to use `x-k8s.io`.

https://github.com/kubernetes/autoscaler/blob/8c35e0aa4bdbab9caf03f8b8926d94f321c6e9a4/cluster-autoscaler/provisioningrequest/apis/autoscaling.x-k8s.io/v1beta1/register.go#L27-L28

Reply:
This is part of the kep and used by the autoscaler.
https://github.com/kubernetes-sigs/kueue/blob/91d5f645a326dae1ea8542879adcc62ba4db7f5d/keps/1136-provisioning-request-support/README.md?plain=1#L95-L99

We can make it configurable in the future, if needed.

Reply:
There are no plans to change it.

Reply:
thanks @kisieland